Source/StorageUtils.swift

```diff
@@ -0,0 +1,83 @@
+/*   Copyright 2018-2019 Prebid.org, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Foundation
+
+class StorageUtils {
```

Comment:
Should we call this PrivacyUtils?

Reply:
`StorageUtils` is a general utility class to work with persistent storage. It will allow us to expand this class in future without generating a lot of small classes. If we had several big logic entities I would create `PrivacyUtils`, `StorageUtils` etc.

Please keep it as is and we will change it when it will be necessary